letsencrypt/tests/cli_test.py

```diff
@@ -112,9 +111,37 @@ def test_help(self):
         self.assertTrue("--key-path" not in out)
 
         out = self._help_output(['-h'])
-        from letsencrypt import cli
         self.assertTrue(cli.usage_strings(plugins)[0] in out)
 
+
+    @mock.patch('letsencrypt.cli.sys.stdout')
+    @mock.patch('letsencrypt.cli.sys.stderr')
+    @mock.patch('letsencrypt.cli.client.acme_client.Client')
+    @mock.patch('letsencrypt.cli._determine_account')
+    @mock.patch('letsencrypt.cli.client.Client.obtain_and_enroll_certificate')
+    @mock.patch('letsencrypt.cli._auth_from_domains')
+    def test_user_agent(self, _afd, _obt, det, _client, _err, _out):
```

Comment:
nitty nit that you probably shouldn't take the time to fix: You could create a version of `_call` that doesn't mock `letsencrypt.cli.client` and have `_call` use that version internally.